vulkaninfo/vulkaninfo.cpp

```diff
@@ -527,25 +540,28 @@ void GpuDumpFeatures(Printer &p, AppGpu &gpu) {
 }
 
 void GpuDumpFormatProperty(Printer &p, VkFormat fmt, VkFormatProperties prop) {
-    if (p.Type() == OutputType::text) {
-        p.ObjectStart("Properties");
-    } else if (p.Type() == OutputType::html) {
-        p.SetTitleAsType().ObjectStart(VkFormatString(fmt));
-    } else if (p.Type() == OutputType::json) {
-        p.ObjectStart("");
-    }
     if (p.Type() == OutputType::html || p.Type() == OutputType::text) {
+        if (p.Type() == OutputType::html)
```

Comment:
Another opportunity for separate cases in a switch

Reply:
Added